<launch>
    <!-- 기본 정보 노드 -->
    <node pkg="scenario" type="mgeo_pub.py" name="mgeo_pub"  />
    <node pkg="scenario" type="gpsimu_parser.py" name="gpsimu_parser" />

    <!-- 라이다 -->
    <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch"/>
    <!-- <include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/> -->

    <node pkg="scenario" type="lidar_velodyne_cluster.py" name="lidar_velodyne_cluster"  />
    <node pkg="scenario" type="lidar_velodyne_cluster_viz.py" name="lidar_velodyne_cluster_viz"  />

    <node pkg="scenario" type="recong.py" name="recong_pub" output="screen" />

    <!-- <node pkg="scenario" type="global_path_pub.py" name="global_path_pub"  /> -->
    <!-- <node pkg="scenario" type="mgeo_dijkstra_path.py" name="mgeo_dijkstra_path"  /> -->

    <!-- lane_change     -->
    <node pkg="scenario" type="lane_change.py" name="lane_change" args="/Object_topic_to_lidar"  />

    <!-- <node pkg="scenario" type="local_path_pub.py" name="local_path_pub" />     -->
    <node pkg="scenario" type="tf_pub.py" name="tf"  />

    <!-- lattice -->
    <node pkg="scenario" type="lattice_planner.py" name="lattice_planner" output="screen" args="/Object_topic_to_lidar"/>
    
    <!-- image_lane_fitting -->
    <!-- <node pkg="scenario" type="image_lane_fitting.py" name="image_lane_fitting" /> -->

    <!-- purepursuit -->
    <!-- <node pkg="scenario" type="advanced_purepursuit.py" name="advanced_purepursuit"  args="/lattice_path"/> -->

    <!-- ACC -->
    <node pkg="scenario" type="acc.py" name="ACC" args="/lattice_path" output="screen" />
    <!-- <node pkg="scenario" type="acc.py" name="ACC" args="/local_path" output="screen" /> -->

    <!-- rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find scenario_1)/rviz/lane_detection_rviz.rviz" />
    <node pkg="tf" type="static_transform_publisher" name="tf_lidar" args="2 0 1.25 0 0 0 1 Ego velodyne 100"/>
</launch>